examples/hackernews_js_fetch/src/lib.rs

```diff
@@ -1,40 +1,67 @@
-use leptos::{component, view, IntoView};
-use leptos_meta::*;
-use leptos_router::*;
+use leptos::prelude::*;
 mod api;
-pub mod error_template;
-#[cfg(feature = "ssr")]
-pub mod fallback;
 mod routes;
+use leptos_meta::{provide_meta_context, Link, Meta, MetaTags, Stylesheet};
+use leptos_router::{
+    components::{FlatRoutes, Route, Router, RoutingProgress},
+    ParamSegment, StaticSegment,
+};
 use routes::{nav::*, stories::*, story::*, users::*};
-use wasm_bindgen::prelude::wasm_bindgen;
+use std::time::Duration;
+
+pub fn shell(options: LeptosOptions) -> impl IntoView {
+    view! {
+        <!DOCTYPE html>
+        <html lang="en">
+            <head>
+                <meta charset="utf-8"/>
+                <meta name="viewport" content="width=device-width, initial-scale=1"/>
+                <AutoReload options=options.clone()/>
+                <HydrationScripts options/>
+                <MetaTags/>
+            </head>
+            <body>
+                <App/>
+            </body>
+        </html>
+    }
+}
 
 #[component]
 pub fn App() -> impl IntoView {
     provide_meta_context();
+    let (is_routing, set_is_routing) = signal(false);
+
     view! {
-        <Link rel="shortcut icon" type_="image/ico" href="/public/favicon.ico"/>
-        <Stylesheet id="leptos" href="/public/style.css"/>
-        <Meta name="description" content="Leptos implementation of a HackerNews demo."/>
-        <Router>
-            <Nav />
+        <head>
```

Comment:
This extra `<head>` tag should be removed -- you create the `<head>` in the `shell` above.

Reply:
Could you please confirm that it is working now?